packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -134,6 +134,31 @@ class ConfigAccountTransfer {
     }
 }
 
+class ConfigWithdraw {
+    @observable container = 'withdraw';
+    @observable crypto_amount = '';
+    @observable fiat_amount = '';
+    @observable insufficient_fund_error = '';
+    @observable is_timer_visible = false;
+    @observable error = new ConfigError();
+    @observable verification = new ConfigVerification();
+
+    @action.bound
```

Comment:
@farrah-deriv We will use this converter in both `withdraw` and `account_transfer`, I believe it's better to have these 3 methods in the cashier_store itself, instead of having it in `ConfigWithdraw`

Reply:
noted bahar